lib/main.dart

```diff
@@ -105,3 +111,272 @@ class _NavBarState extends State<NavBar>
     );
   }
 }
+
+class HomePage extends StatelessWidget {
```

Comment:
Would you be able to move the HomePage into another file in a pages/ directory?

Reply:
Alright, I've pushed those changes.